### ISSUE TITLE
Fractional sizes of container

### ISSUE THREAD

**Issue by indapublic** [select2/select2#3558]
Hello. Thank you for your great select2!!

I have some issues if I have fractional sizes of container [example](https://dl.dropboxusercontent.com/spa/f2flw5cwyoai67g/ehpty7jm.png)

Inspecting your code, I guess you should to use something like

```
width = container[0].getBoundingClientRect().width
```

instead of

```
width = container.outerWidth(false)
```

Isn't it?

**Comment by kevin-brown** [comment]
Right now [`getBoundingClientRect`](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect) is not used because it is lacking the `width` property in IE 8.

I'm not opposed to a pull request which tries `getBoundingClientRect` and falls back to `$.fn.width` if the `width` isn't included.

**Comment by Daijobou** [comment]
Sorry, I not found this issue. But maybe this helps:
https://github.com/zeroclipboard/zeroclipboard/commit/be4573579f9dbccad4280bd06e2e72c896fa5c66
and here the reason:
http://stackoverflow.com/a/17686282

**Comment by Daijobou** [comment]
Issue still exist in version 4.0.3 :)

**Comment by alexweissman** [comment]
Duplicate of #1692, fixed by #4471.
